autoapi/_objects.py

```diff
@@ -408,12 +408,18 @@ class PythonModule(TopLevelPythonPythonMapper):
 
     type = "module"
 
+    def _should_skip(self) -> bool:
+        return False
```

Comment:
It's legitimate for a `TopLevelPythonPythonMapper` to be skipped when `self.obj.get("hide", False)` or when it's private. Perhaps it deserves an option separate from `"private-members"`, but for now at least it's right that option to control when a module/package is rendered.

Reply:
@AWhetter Fair point, I added some minor changes to at least support "hide" and "private" as you suggested